mypy/server/deps.py

```diff
@@ -506,6 +537,13 @@ def visit_yield_from_expr(self, e: YieldFromExpr) -> None:
 
     # Helpers
 
+    def add_type_alias_deps(self, target: str, add_trigger: bool = False) -> None:
```

Comment:
The `add_trigger` argument name is unclear. Can you come up with another name? Maybe something about triggering recursively?